project-maintainers.csv

```diff
@@ -99,6 +99,7 @@ Graduated,containerd,Michael Crosby,Apple,crosbymichael,https://github.com/conta
 ,,Kevin Parsons,Microsoft,kevpar,
 ,,Kazuyoshi Kato,Fly.io,kzys,
 ,,Samuel Karp,Google,samuelkarp,
+,,Kirtana Ashok,Microsoft,kirtana.ashok@gmail.com,

```

Comment:
The final column here should be a GitHub handle rather than email address: kiashok